django_mesh_test_project/settings.py

```diff
@@ -89,12 +90,18 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.6/howto/static-files/
 
-STATIC_URL = '/static/'
 STATIC_PATH = os.path.join(PROJECT_ROOT, 'static')
-
+STATIC_URL = '/static/'
 STATICFILES_DIRS = (
-     STATIC_PATH,
-    )
+    STATIC_PATH,
+)
+
+MEDIA_PATH = os.path.join(PROJECT_ROOT, 'media')
+MEDIA_URL = '/media/'
+MEDIAFILES_DIRS = (
```

Comment:
I don't think this setting is needed, looks like it's part of this package: https://github.com/nai-central/django-mediafiles/